tests/run_tests.sh

```diff
@@ -859,7 +859,7 @@ dtbs_equal_tests () {
 fdtget_tests () {
     dts=label01.dts
     dtb=$dts.fdtget.test.dtb
-    run_dtc_test -O dtb -o $dtb "$SRCDIR/$dts"
+    run_dtc_test -@ -O dtb -o $dtb "$SRCDIR/$dts"
```

Comment:
Obviously you need to add `-@` to test the new functionality.  However, this means you're no longer testing the old functionality on a tree that's not compiled with `-@`.  I'd prefer to extend this so that both the `-@` and not `-@` cases are exercised.

Reply:
Good point. I'll see if I can figure out an easy way to do these tests both ways, and then either omit the new tests when not built with -@, or turn them into expected errors.